src/format/mod.rs

```diff
@@ -75,6 +75,10 @@ pub enum Numeric {
     /// Full Gregorian year (FW=4, PW=∞).
     /// May accept years before 1 BCE or after 9999 CE, given an initial sign.
     Year,
+    /// Full Gregorian year (FW=2, PW=∞). Can be negative.
+    ///
+    /// Prints last two digits for year >= 1000 or year <= -1000.
```

Comment:
Could you explicitly mention how ambiguous this is? E.g. that the years `1120` and `2020` will print the same year? And that it errors in years outside of the mentioned range?

Also, why does this error for N `-1000 < N < 1000`?